src/DynamoRevit/ViewModel/DynamoRevitViewModel.cs

```diff
@@ -46,6 +45,14 @@ public static DynamoRevitViewModel Start(StartConfiguration startConfiguration)
                     throw new Exception("An instance of RevitDynamoViewModel is required to construct a DynamoRevitViewModel.");
```

Comment:
I know this is unrleated- but isn't this exception incorrect should it say ```("An instance of RevitDynamoModel is required to construct a DynamoRevitViewModel.");```

?